.drone.star

```diff
@@ -561,6 +601,7 @@ def phptests(testType):
 		'logLevel': '2',
 		'cephS3': False,
 		'scalityS3': False,
+		'externalTypes': ['none'],
```

Comment:
I am not sure why this `externalTypes` stuff needs to be added. (and the loop for it below). The apps do not currently use it.

But it will not break anything, the code will just always follow the path for `none`

Reply:
I added them as they were in `core/.drone.star`. Since, they are not needed for apps I'll just remove them